backend/typescript/rest/campRoutes.ts

```diff
@@ -0,0 +1,44 @@
+import { Router } from "express";
+import ICampService from "../services/interfaces/campService";
+import CampService from "../services/implementations/campService";
+import { getErrorMessage } from "../utilities/errorUtils";
+import { createCampDtoValidator } from "../middlewares/validators/campValidators";
+
+const campRouter: Router = Router();
+
+const campService: ICampService = new CampService();
+
+campRouter.get("/csv/:id", async (req, res) => {
+  try {
+    const csvString = await campService.generateCampersCSV(req.params.id);
+    res.status(200).json(csvString);
+  } catch (error: unknown) {
+    res.status(500).json({ error: getErrorMessage(error) });
+  }
+});
+
+/* Create a camp */
+campRouter.post("/", createCampDtoValidator, async (req, res) => {
+  try {
+    const newCamp = await campService.createCamp({
+      campers: req.body.campers,
+      waitlist: req.body.waitlist,
```

Comment:
hm I think we can remove these since admins won't be specifying campers or waitlisted campers at camp creation